---
title: >
  The retrovirus conundrum
subtitle: >
  How do retroviruses insert their genome into host DNA?
date: "2010-11-18 20:15:42 +0000"
image: "http://felixonline.co.uk/"
headline: true
featured: true
imported: true
aliases:
 - /science/386/the-retrovirus-conundrum
comments:
categories:
 - science
tags:
 - imported
authors:
 - vbb08
highlights:
---

Imperial College researchers from the Department of Infectious Diseases have just succeeded in determining the mechanism of retroviral integration, using x-rays to analyse the key molecular structures involved in this process.

The Diamond synchrotron in Oxfordshire was the researchers’ main tool to produce intense X-ray beams enabling them to observe molecular interactions. “This kind of fundamental research is vital if we are to advance our understanding of the viruses and diseases that affect millions of people around the world. Knowing the 3D structure of the mechanisms involved is like being able to see inside the engine of your car. If you can actually see what is happening, you get an idea of how you can fix it”, says Professor Thomas Sorensen, who is Principal Beamline Scientist at the Diamond synchrotron.

A retrovirus relies on its host-cell machinery to replicate its genome. The latter is in the form of RNA, and is converted to DNA once in the host cell. Viral DNA must then be inserted within host-cell chromosomal DNA. The way this insertion is carried out has, up till now, been a biological enigma. The Imperial College researchers have managed to describe a key agent in this vital step for the virus: the intasome, a nucleoprotein complex composed of an integrase tetramer (IT). Integrase is a formerly known enzyme which confines and fuses the ends of viral and host DNA together.

Prototype foamy virus (PFV) is very similar to HIV in its way of integrating its genome. With the synchrotron, the authors analysed crystal structures of host DNA in complex with the intasome of PFV. That is, the 3D structures of integrase bound to both viral and host DNA in atomic detail. From this observation, the authors elucidated the confinement mechanism of target DNA before integration, as well as the intermediary steps of post-catalytic strand transfer. “Only 18 months ago we had a rather sketchy understanding of retroviral integration”, says Dr Peter Cherepanov from the Department of Medicine.

Two integrase tetramers (ITs) come together to form a narrow cleft, which accommodates the host DNA in a highly bent configuration, facilitating access for IT active sites to the scissile phosphodiester bonds. For insertion into chromatin, the ITs are thought to interact with host nucleosomal DNA , and/or the histone octamer. Once this is performed, the phosphodiester linking viral and host DNA gets ejected out of the active site. As expected from knowledge on the relatively low degree of sequence selectivity of retroviruses for chromosomal DNA, interactions between IT and host DNA bases are sparse. Indeed, strong selectivity for chromosomal DNA would limit possible integration sites and thus reduce viral fitness. Yet two sites of close contact were identified.

In the context of gene-therapy, Cherepanov stresses that: “one of the main problems with the current method is that retroviral integration is too random. [...] Ideally, we want to insert therapeutic genes in predefined, safe locations of the human genome”. With these findings, one could theoretically create site-specific retroviral vector systems by designing an integrase with higher selectivity for a wanted target DNA sequence.

The synthetic retrovirus could insert a functional copy of a desired gene into a human chromosome by greater certainty of insertion site. This greatly improves the technique’s safety by for instance reducing the risk of activating an oncogene, thereby avoiding the reoccurrence of former cases such as leukaemia in treated patients. The discoveries would also help improving existing antiviral strategies, facilitate the design of better drugs to combat AIDS, and stimulate novel approaches to blocking viral replication.
