---
title: >
  New insight into gene expression
subtitle: >
  New genome sequencing methods are put into use
date: "2012-01-26 22:27:38 +0000"

# Attributes from Felix Online V1
id: "2063"
old_path: /science/2063/new-insight-into-gene-expression
aliases:
 - /science/2063/new-insight-into-gene-expression
imported: true
comments:

# Article Taxonomies
categories:
 - science
tags:
 - imported
 - image
 - multi-author
authors:
 - ak6309
 - ps3110
highlights:

# Homepage control params
headline: true
featured: true
image: "http://felixonline.co.uk/img/upload/201201262228-ak6309-39840_web.jpg"
image_caption: ""
---

A new genomic sequencing method has been put into practice by Professor Franklin Pugh and graduate student Ho Sung Rhee, after development in Penn State University. The method, known as ChIP-exo, gives us an insight into the proteins that read and regulate chromosomes (transcription preinitiation complexes, or PICs) and was detailed in a paper published online in Nature.
 Although it varies in proportion from species to species, a lot of DNA does not actually code for anything; around 98% of a human genome is non-coding, compared to 11% of the genome of bacterium E. coli. This means that the coding portion that makes up genes has to be identified, and, with the human genome consisting of 3.2 billion base pairs, this can prove tricky. Genes themselves have integral sequences called promoter regions that allow their identification in between all the millions of non-coding base pairs, and ChIP-exo takes advantage of these integral sequences as PICs bind to them. Furthermore, ChIP-exo eliminates “transcriptional noise,” unwanted levels of transcription to allow detection of more PIC binding sequences than other sequencing methods would allow. “The advantage over other techniques of [ChIP-exo] is its ability to narrow down any binding location across millions and billions of nucleotide genomes to a certainty of about one nucleotide,” said Pugh.
 In the new technique, genes are first transcribed into a more dynamic molecule called RNA in a process called transcription, that occurs in the nucleus. RNA is then translocated to the cytoplasm of the cell to provide the instructions to manufacture the protein. PICs are different proteins that allow and control this RNA manufacture, and bind at the promoter regions of genes in a sequential manner to allow transcription to proceed. A better understanding of how they position themselves along the DNA allows a greater insight into where the genes actually start in the genome, and how gene expression in cells is regulated, as not all genes are active all of the time.
 To identify these PICs and the sequences they bind to, exonucleases (the “exo” in ChIP-exo) were used. These enzymes catalyse the degradation of the double stranded helix effectively destroying the genome. Multiple exonucleases were applied across the whole genome, and it degraded the DNA until a cross-linking point was reached in DNA where the PICs begin to interact with the promoter regions. These conserved regions were then isolated and sequenced accordingly. Rhee ran ChIP-exo for various PICS, RNA Polymerase II, the enzyme that actively synthesises RNA, and for GTFs, General Transcription Factors, and mapped of the precise location of more than six thousand of these PICs in yeast. The technique showed how they interacted specifically with nucleosomes: DNA bound tightly to proteins called histones to allow it to be compacted into the nucleus.
 So what does this indicate in the long run? Studying yeast here allows for many of the principles to be extrapolated on to other species, including humans, to allow a better insight into their gene regulation and initiation; a small yet crucial step towards being fully aware of the genomic capabilities of humans and how we can manipulate it in the future. Genetically modified crops, genetic diseases, developmental disorders, and advancing in on the cure for cancer are all areas that can benefit highly from the new data this technique can deliver, and, when used in conjunction with existing methods, the limits to our grasp of genetics have now been extended.
 DOI: 10.1038/nature10799
